lib/src/util/__tests__/vulnerabilitySeverityLevels.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {
+  formatVulnerabilityScore,
+  getVulnerabilitySeverityLevel,
+  VulnerabilityScore
+} from '../vulnerabilitySeverityLevels';
+
+describe('vulnerabilitySeverityLevels utilities', function() {
+  describe('formatVulnerabilityScore', function() {
+    it('returns the correct Vulnerability (CVSS) score string given floating point number', function() {
+      expect(() => formatVulnerabilityScore(-1)).toThrowError('Vulnerability score must be between 0.0 and 10.0.');
+      expect(() => formatVulnerabilityScore(11)).toThrowError('Vulnerability score must be between 0.0 and 10.0.');
+
+      expect(() => formatVulnerabilityScore(7.00000000002))
+          .toThrowError('Vulnerability score must be one decimal place.');
+      expect(() => formatVulnerabilityScore(4.55)).toThrowError('Vulnerability score must be one decimal place.');
+      expect(() => formatVulnerabilityScore(5.88)).toThrowError('Vulnerability score must be one decimal place.');
+
+      expect(formatVulnerabilityScore(0)).toBe('0.0');
+      expect(formatVulnerabilityScore(2)).toBe('2.0');
+      expect(formatVulnerabilityScore(3.3)).toBe('3.3');
+      expect(formatVulnerabilityScore(10)).toBe('10.0');
+    });
```

Comment:
Test `-0`, `Nan`, `Infinity` and `-Infinity` too.  `-0` is a distinct value in floating point math.

Reply:
> Test `-0`, `NaN`, `Infinity` and `-Infinity` too. `-0` is a distinct value in floating point math.

Thanks, updated test here: [9db49e1](https://github.com/sonatype/sonatype-react-shared-components/pull/1202/commits/9db49e1bd28365bca5f298c7b6467e59a92d6a2c)